staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -129,21 +128,6 @@ func (IntOrString) OpenAPISchemaType() []string { return []string{"string"} }
 // the OpenAPI spec of this type.
 func (IntOrString) OpenAPISchemaFormat() string { return "int-or-string" }
 
-func (intstr *IntOrString) Fuzz(c fuzz.Continue) {
```

Comment:
metav1.{MicroTime,Time} both implement this as well... same thing needed there?

Reply:
yeah, I held on that for feedback on the tag.

Reply:
done.